src/io/configurationio.js

```diff
@@ -5,8 +5,11 @@ const defaultConfigData = {
   sertopPath: '',
   serapiVersion: '',
   libraryVersion: '',
+  zoom: '',
+  theme: '',
 };
-const possibleKeys = ['sertopPath', 'serapiVersion', 'libraryVersion'];
+const possibleKeys = [
```

Comment:
This is annoying to have to add for every setting we should probably change this.
Out of scope for this PR though.